src/sim.jl

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env julia
-using Distributions
+# using Distributions
```

Comment:
Have you dropped used of Distributions.jl entirely? If so, should drop this.

Reply:
Put it back in.